src/main/webapp/jsp/instructorFeedbacks.jsp

```diff
@@ -686,7 +686,8 @@
                 }
             %>
         </table>
-        <br> <br> <br>
+		<p align=center>Note: The table above does not contain sessions from archived courses. To view sessions from an archived course, unarchive the course first.</p>
```

Comment:
It might be preferable if the 'align' attribute isn't used. Seems like it's an obsolete api which may not be supported in future, from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p#Attributes

We have a "align-center" css class that you use here instead.